app/views/layouts/_lte_admin_sidebar.html.erb

```diff
@@ -85,7 +86,8 @@
     <a href="#" class="nav-link <%= active_class(['purchases']) %>">
```

Comment:
I am curious. Why do we use `active_class(['purchases'])` above for Users?

Reply:
I'm guessing that was a copy and paste issue. Should probably be users.